From f3571bee81cd569767ccac6f25499cb54455c6d5 Mon Sep 17 00:00:00 2001
From: Siva Durga Prasad Paladugu <siva.durga.paladugu@xilinx.com>
Date: Tue, 22 Jan 2019 14:34:12 +0530
Subject: [PATCH] arm: zynq: Enable CONFIG_REMAKE_ELF

This patch enables CONFIG_REMAKE_ELF for Zynq platform
so that it generates u-boot.elf from binary which works
for all Zynq boards with OF_SEPARATE option enabled.

Upstream-Status: Backport [https://github.com/u-boot/u-boot.git]

Signed-off-by: Siva Durga Prasad Paladugu <siva.durga.paladugu@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Ovidiu Panait <ovpanait@gmail.com>

---
 include/configs/zynq-common.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/zynq-common.h b/include/configs/zynq-common.h
index a38ca05e0d..4320eb46ef 100644
--- a/include/configs/zynq-common.h
+++ b/include/configs/zynq-common.h
@@ -15,6 +15,8 @@
 # define CONFIG_CPU_FREQ_HZ	800000000
 #endif
 
+#define CONFIG_REMAKE_ELF
+
 /* Cache options */
 #define CONFIG_SYS_L2CACHE_OFF
 #ifndef CONFIG_SYS_L2CACHE_OFF
